From 5a84f7feef512d08e49bd54f9bbdd7cbb1b7931d Mon Sep 17 00:00:00 2001
From: Yu YongZhen <yuyz@rock-chips.com>
Date: Fri, 23 Aug 2019 06:55:49 +0000
Subject: [PATCH] clk/rockchip: rk618: enlarge buffer size for clk name

strlcpy(clkin_name, __clk_get_name(clk), sizeof(clkin_name));
the clkin_name will wrong when clk is clk_i2s0_8ch_tx_out

Change-Id: If5603b5c31d4705289d075b2b375a35f3d4e4178
Signed-off-by: Yu YongZhen <yuyz@rock-chips.com>
---
 drivers/clk/rockchip/rk618/clk-rk618.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/clk/rockchip/rk618/clk-rk618.c b/drivers/clk/rockchip/rk618/clk-rk618.c
index 6f4d264292e8..d2e6ad2bbfd5 100644
--- a/drivers/clk/rockchip/rk618/clk-rk618.c
+++ b/drivers/clk/rockchip/rk618/clk-rk618.c
@@ -194,9 +194,9 @@ struct rk618_cru {
 	struct clk_onecell_data clk_data;
 };
 
-static char clkin_name[16] = "dummy";
-static char lcdc0_dclkp_name[16] = "dummy";
-static char lcdc1_dclkp_name[16] = "dummy";
+static char clkin_name[32] = "dummy";
+static char lcdc0_dclkp_name[32] = "dummy";
+static char lcdc1_dclkp_name[32] = "dummy";
 
 #define PNAME(x) static const char *const x[]
 
-- 
2.35.3

